dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java

```diff
@@ -258,6 +258,14 @@ private Constants() {
     public static final String YYYY_MM_DD_HH_MM_SS = "yyyy-MM-dd HH:mm:ss";
 
 
+
```

Comment:
Remove the extra blank lines and only keep one blank line.

BTW, @zhongjiajie can you please set up a code style rule to only allow at most one blank line between member fields / methods?

Reply:
👌 , would try today